src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/types';
+import { getUser } from '../../utils/getUser';
```

Comment:
Avoid using the `getUser` function directly in this component. Instead, prepare the user data in the `App` component and pass it down as props. This aligns with the checklist item: 'Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components.'